vortexasdk/endpoints/cargo_movements.py

```diff
@@ -73,6 +74,7 @@ def search(
         exclude_vessel_propulsion: Union[ID, List[ID]] = None,
         disable_geographic_exclusion_rules: bool = None,
         intra_movements: str = None,
+        quantity_at_time_of: str = None,
```

Comment:
Just to make sure - don't we need any default value?

Reply:
And do we need it for search endpoint or TS endpoint only?

Reply:
yes, you're right, `quantity_at_time_of is optional` and default is 'load'. Provided that
As for search endpoint - this cargo movement is already a search endpoint, or I mixing something up?